fujiji-client/src/components/NavBar/NavBar.jsx

```diff
@@ -1,115 +1,65 @@
 import PropTypes from 'prop-types';
-import {
-  Box,
-  Flex,
-  Avatar,
-  Link,
-  Button,
-  Menu,
-  MenuButton,
-  MenuList,
-  MenuItem,
-  MenuDivider,
-  Stack,
-} from '@chakra-ui/react';
```

Comment:
I was wondering if we want to use Chakra-UI component instead of making a component from scratch by ourselves?

If we use the Chakra-UI we don't really need a CSS module (as we can use CSS in JS instead) and we will have the same and consistent design across our components (same font, color scheme, media query, etc..).

Reply:
Chakra-UI already have menu built-in: https://chakra-ui.com/docs/overlay/menu